trino/dbapi.py

```diff
@@ -321,7 +321,8 @@ def rowcount(self):
         operation. For example, the result of a SELECT query is streamed and
         the number of rows is only knowns when all rows have been retrieved.
         """
-
+        if self._query is not None and self._query.update_count is not None:
+            return self._query.update_count
         return -1
```

Comment:
This is not obvious so maybe a link to https://peps.python.org/pep-0249/#rowcount here would be useful i.e. the -1 is what the DB-API requires.